sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -158,5 +159,22 @@ class DDLSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-14129: alter table - rename") {
+    import testImplicits._
+
+    withTempPath { dir =>
+      val path = dir.getCanonicalPath
+      val df =
+        sparkContext.parallelize(1 to 10).toDF("num")
+      df.write.format("parquet").save(path)
+      // create the temporary table
+      sql(s"CREATE TEMPORARY TABLE test1 USING parquet OPTIONS (path '$path')")
+      assert(sql("SHOW TABLES").collect().count(_ == Row("test1", true)) == 1)
```

Comment:
This is quite hard to follow. Please use `checkAnswer(sql(...), ...)` here and everywhere else (see the other tests in this file for examples).